src/style-auro-calendar-cell.scss

```diff
@@ -122,7 +122,7 @@
 }
 
 ::slotted([slot^="date_"]) {
-  color: var(--ds-color-text-link-default, $ds-color-text-link-default);
+  color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
```

Comment:
Per the design, we need to keep the token we are using for the primary use case of the date slot. Please revert this change. If there were instructions from a different designer, please let me know and I can discuss this with my team!

Reply:
The changes to the default color of the date price slot has been approved by designers on the Auro team.